docs/pt/docs/tutorial/debugging.md

```diff
@@ -0,0 +1,112 @@
+# Debugando
+
+Você pode conectar o debugger no seu editor, por exemplo com o Visual Studio Code ou PyCharm.
+
+## Chamar `uvicorn`
+
+Na sua aplicação FastAPI, importe e rode `uvicorn` diretamente:
+
+```Python hl_lines="1  15"
+{!../../../docs_src/debugging/tutorial001.py!}
+```
+
+### Sobre `__name__ == "__main__"`
+
+O principal propósito do `__name__ == "__main__"` é ter um código que é executado quando o seu arquivo é chamado com:
+
+<div class="termy">
+
+```console
+$ python myapp.py
+```
+
+</div>
+
+mas não é chamado quando outro arquivo o importa, como em:
+
+```Python
+from myapp import app
+```
+
+#### Mais detalhes
+
+Digamos que o seu arquivo tem o nome de `myapp.py`.
+
+Se você rodá-lo com:
+
+<div class="termy">
+
+```console
+$ python myapp.py
+```
+
+</div>
+
+a variável interna `__name__` no seu arquivo, criada automaticamente pelo Python, vai ter como valor a string `"__main__"`.
+
+Então, a sessão:
+
+```Python
+   uvicorn.run(app, host="0.0.0.0", port=8000)
+```
+
+vai rodar.
+
+---
+
+Isso não vai acontecer se você importar aquele módulo (arquivo).
+
+Então, se você tiver outro arquivo `importer.py` com:
+
+```Python
+from myapp import app
+
+# Some more code
+```
+
+nesse caso, a variável automática dentro de `myapp.py` não terá a variável `__name__` com o valor de `"__main__"`.
+
+Então, a linha:
+
+```Python
+   uvicorn.run(app, host="0.0.0.0", port=8000)
+```
+
+não será executada.
+
+!!! info "Informação"
```

Comment:
I believe the `"Informação"` isn't really necessary here. It's adding nothing - actually, it isn't even in the original language, and it's redundant. 🤔

```suggestion
!!! info
```